SPECS-EXTENDED/rubygem-semantic_puppet/rubygem-semantic_puppet.spec

```diff
@@ -8,30 +8,31 @@ Version:        1.0.4
 Release:        1%{?dist}
 Group:          Development/Languages
 License:        MIT
-Vendor:		Microsoft Corporation
+Vendor:         Microsoft Corporation
 Distribution:	Mariner
-URL:            https://rubygems.org/gems/%{gem_name}/versions/%{version}
-Source0:        https://rubygems.org/downloads/%{gem_name}-%{version}.gem
+URL:            https://github.com/puppetlabs/semantic_puppet
+#Source0:        https://github.com/puppetlabs/semantic_puppet/archive/refs/tags/%{version}.tar.gz
+Source0:        %{gem_name}-%{version}.tar.gz
+BuildRequires:  git
 BuildRequires:  ruby
 
 %description
 Tools used by Puppet to parse, validate, and compare Semantic Versions and
 Version Ranges and to query and resolve module dependencies.
 
 %prep
-%setup -q -c -T
+%setup -q -n %{gem_name}-%{version}
 
 %build
+gem build %{gem_name}
 
 %install
-gem install -V --local --force --install-dir %{buildroot}/%{gemdir} %{SOURCE0}
+gem install -V --local --force --install-dir %{buildroot}/%{gemdir} %{gem_name}-%{version}.gem
 
 %files
 %defattr(-,root,root,-)
-%license %{gemdir}/gems/%{gem_name}-%{version}/LICENSE
```

Comment:
Is there no license information at all in the new source now? We'd need something. If there's none, we should add a `Source` file explaining how we know the license for this package and include that files through the `%license` macro.

Look for "LICENSE.PTR" in other specs to see examples.

Reply:
Added license file back from source.